lib/internal/cluster/round_robin_handle.js

```diff
@@ -93,21 +90,24 @@ RoundRobinHandle.prototype.remove = function(worker) {
 
 RoundRobinHandle.prototype.distribute = function(err, handle) {
   this.handles.push(handle);
-  const worker = this.free.shift();
+  const workerId = this.free.keys().next().value;
```

Comment:
I think it would be better to just use `.entries().next().value` so that `workerId` and `worker` can be obtained in one go.

Reply:
Done 👍